parent/pom.xml

```diff
@@ -1917,6 +1920,16 @@
                 <artifactId>stax-ex</artifactId>
                 <version>${version.stax.ex}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <version>${httpcomponents-httpclient.wso2.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.orbit.org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>${commons-lang3.orbit.version}</version>
+            </dependency>
```

Comment:
I think commons lang library should already be available. Do we need need lang3 APIs for the ArrayUtils ops? Wouldn't the existing ones be enough?